dynamiqs/mesolve/rouchon.py

```diff
@@ -54,33 +54,34 @@ def Ms(self, Hnh: Tensor, fwd: bool = True) -> tuple(Tensor, Tensor):
     def T(self, R: Tensor) -> Tensor:
         # we normalize the map at each time step by inverting `R` using its Cholesky
         # decomposition `R = T @ T.mT`
-        # -> (b_H, 1, n, n)
+        # -> (b_H, 1, 1, n, n)
```

Comment:
`M0` is batched over `L`, so `R(M0)` and `T(R)` are as well. Can you check the dimensions you wrote are consistent with what's actually being returned by the function?

Reply:
Nice catch!